### PR TITLE
Add example script with test data

### PR DESCRIPTION
**Pull request by dstansby**
This should make it way easier for us to test any changes we make to `arboretum` by adding a simple script (`examples/show_sample_data.py`) that can be run to launch a `napari` window with a segmenation and tracks already loaded.

Fixes https://github.com/lowe-lab-ucl/arboretum/issues/30

### CODE REVIEW

**Comment by dstansby** [comment]
This should be changed to use data at https://github.com/lowe-lab-ucl/btrack-examples, so converting back to draft.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 napari>=0.4.0
 numpy>=1.17.3
+pooch>=1
 pyqtgraph>=0.10.0
+scikit-image
```
[the remark this answers is not in the document]
Comment (dstansby) [reply requirements.txt:5]: It's only used for `imread`, which I imagine isn't a very new feature in `scikit-image`, so I don't think it's worth the time trying to work out what to pin this to.

File: napari_arboretum/sample/sample_data.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    Load some sample data.
+
+    Returns
+    -------
+    tracks : napari.layers.Tracks
+    segmentation : napari.layers.Labels
+    """
+    # Load track data files
+    track_data = pd.read_csv(POOCH.fetch("examples/tracks.csv", progressbar=True))
+    properties = pd.read_csv(POOCH.fetch("examples/properties.csv", progressbar=True))
```
[the remark this answers is not in the document]
Comment (dstansby) [reply napari_arboretum/sample/sample_data.py:30]: I'm confused, this is OS agnostic? The paths on the right are (relative) URLs.

File: examples/show_sample_data.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+viewer = napari.Viewer()
+viewer.add_layer(track)
+viewer.add_layer(segmentation)
+viewer.window.add_plugin_dock_widget(
+    plugin_name="napari-arboretum", widget_name="Arboretum"
+)
+
+if __name__ == '__main__':
+    napari.run()
```
[the remark this answers is not in the document]
Comment (dstansby) [reply examples/show_sample_data.py:24]: 👍 I'll add an explanatory comment to the script here

File: napari_arboretum/sample/sample_data.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dstansby) [reply napari_arboretum/sample/sample_data.py:37]: It's a URL, not a path, so it doesn't matter which OS is being used.

[the remark this answers is not in the document]
Comment (paddyroddy) [reply napari_arboretum/sample/sample_data.py:37]: Okay

**Pull request merged**